app-crypt/veracrypt/veracrypt-1.19.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit flag-o-matic linux-info pax-utils toolchain-funcs wxwidgets
```

Comment:
Per https://devmanual.gentoo.org/eclass-reference/eutils.eclass/index.html, the use of `newicon` requires you to inherit `eutils`